OrangeHRMTests/PageObjects/Elements/Buttons.cs

```diff
@@ -0,0 +1,39 @@
+﻿using OpenQA.Selenium;
+using OrangeHRMTests.Common.WebElements;
+
+namespace OrangeHRMTests.PageObjects.Elements
+{
+    public static class Buttons
```

Comment:
Not really good usage of pattern
I would better create every button on Page Object level. If you have buttons that are available on every page - move it to BasePage